Sources/MapboxCoreNavigation/ReroutingObserverDelegate.swift

```diff
@@ -0,0 +1,33 @@
+import Turf
+import MapboxDirections
+import MapboxNavigationNative
+
+
+protocol ReroutingObserverDelegate: AnyObject {
```

Comment:
Based on the names and argument types of the methods in this protocol, wouldn’t this be a RerouteControllerDelegate?